client/specs/Map.test.jsx

```diff
@@ -0,0 +1,34 @@
+import * as enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, mount, render } from 'enzyme';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-dom/test-utils';
+import sinon from 'sinon';
+import Googlemap from '../src/Map.jsx';
+
+enzyme.configure({ adapter: new Adapter() });
+
+describe('Map test', () => {
+
+ //need to change it to false
```

Comment:
Opportunity to remove commented out text. If you're still working on this, its fine to leave it with me.